### PR TITLE
feature/AT-UN-UI_16_Migration

### PR DESCRIPTION
**Pull request by BrendaMorales9719**

### CODE REVIEW

File: src/views/layouts/main.hbs [hunk 1]
```diff
@@ -1,28 +1,114 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8">
-    <title>AT University</title>
-    <!-- BOOTSTRAP - BOOTSWATCH THEME - COSMOS -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootswatch/4.5.2/materia/bootstrap.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>AT ::: UI</title>
```
Comment (at-internship) [inline src/views/layouts/main.hbs]: AT ::: UNIVERSITY

File: src/views/partials/navigation.hbs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <a class="navbar-brand" href="/"><img src="/img/at-logo.png" title="AT" width="150"></a>
+
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav"
+            aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav">
+                <li class="nav-item active">
+                    <a class="nav-link" href="/users/home">
```
Comment (at-internship) [inline src/views/partials/navigation.hbs]: We don't have "home" should be `<a class="nav-link" href="/dashboard">`

File: src/views/partials/navigation.hbs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                        <span class="d-none d-lg-block">
+                            <img src="/img/icons/menu-home-icon.png" title="Home" class="icons">
```
Comment (at-internship) [inline src/views/partials/navigation.hbs]: ``

File: src/views/partials/navigation.hbs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+                        </span>
+                        <span class="d-lg-none">Home</span>
+                    </a>
+                </li>
+                {{#if user}}
```
Comment (at-internship) [inline src/views/partials/navigation.hbs:19]: Add Code comment

File: src/views/partials/navigation.hbs [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+                        <span class="d-lg-none">AT University</span>
+                    </a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/at-resources/dashboard">
+                        <span class="d-none d-lg-block">AT Resources</span>
+                        <span class="d-lg-none">AT Resources</span>
+                    </a>
+                </li>
+                {{/if}}
```
Comment (at-internship) [inline src/views/partials/navigation.hbs:32]: Add Code comment

File: src/views/partials/navigation.hbs [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+            </ul>
+
+            <ul class="navbar-nav ml-auto">
```
Comment (at-internship) [inline src/views/partials/navigation.hbs]: For now, comment this section, Add Code comment <! - ->

**Review by at-internship — CHANGES_REQUESTED** [review]

**Review by pinto1997 — APPROVED** [review]

**Review by at-internship — APPROVED** [review]

**Pull request merged**